src/main/java/uk/gov/homeoffice/digital/sas/timecard/validators/timeentry/TimeEntryValidator.java

```diff
@@ -20,6 +20,45 @@ public class TimeEntryValidator implements ConstraintValidator<TimeEntryConstrai
   @Override
   public boolean isValid(TimeEntry timeEntry, ConstraintValidatorContext context) {
 
+    if (timeEntry.getActualEndTime() != null
+        && timeEntry.getActualStartTime().after(timeEntry.getActualEndTime())) {
+      String message = "End time must be after start time";
```

Comment:
nit: we can improve readability by using `var` for variables where the type can be inferred from the value it's initialised with. 
Also on he next line and a few other lines.

Reply:
I think I've caught them now?

Reply:
looks good